ui/server/src/test/scala/pl/touk/nussknacker/ui/api/ManagementResourcesSpec.scala

```diff
@@ -138,8 +139,22 @@ class ManagementResourcesSpec extends FunSuite with ScalatestRouteTest with Fail
 
   test("rejects deploy without comment if comment needed") {
     saveProcessAndAssertSuccess(SampleProcess.process.id, SampleProcess.process)
-    deployProcess(SampleProcess.process.id, true) ~> check {
-      rejection shouldBe server.ValidationRejection("Comment is required", None)
+    deployProcess(SampleProcess.process.id, deploySettings = DeploySettings("requiredCommentPattern", "exampleRequiredComment")) ~> check {
+      status shouldBe StatusCodes.BadRequest
+    }
+  }
+
+  test("accepts deploy with valid comment") {
+    saveProcessAndAssertSuccess(SampleProcess.process.id, SampleProcess.process)
+    deployProcess(SampleProcess.process.id, deploySettings = DeploySettings("[a-zA-Z]*Comment[0-9]*", "exampleComment123"), Some("validComment123")) ~> check {
+      status shouldBe StatusCodes.OK
+    }
+  }
+
+  test("rejects deploy with invalid comment") {
```

Comment:
I think we don't need test invalid / valid comment - because we have already tested it by Unit Test above?

Reply:
ok, you are right that the tests are really similar, removed it